python/fledge/plugins/south/person_detection/person_detection.py

```diff
@@ -451,10 +531,10 @@ def run(self):
                     xmax_model = round(boxes[i][3], 3)
 
                     # map the bounding boxes from model to the window
-                    ymin = int(max(1, (ymin_model * self.camera_width)))
-                    xmin = int(max(1, (xmin_model * self.camera_height)))
-                    ymax = int(min(self.camera_width, (ymax_model * self.camera_width)))
-                    xmax = int(min(self.camera_height, (xmax_model * self.camera_height)))
+                    ymin = int(max(1, (ymin_model * self.camera_height)))
```

Comment:
The coordinates are 1-based or 0-based?

Reply:
It is 0 [based](https://answers.opencv.org/question/35111/origin-pixel-in-the-image-coordinate-system-in-opencv/?answer=35135#post-id-35135)

Reply:
Then, for xmin and ymin, shouldn't it be max(0, ...)? Also "self.camera_height" is being used in 3 expressions above, seems atleast one is incorrect. Please check.

Reply:
I think it is correct. The code is taken from [here](https://github.com/EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi/blob/200a7fa02cf6aa464c5f1e9da61df83f224c85be/TFLite_detection_stream.py#L199) .

Reply:
Given minimum value zero.